packages/mobile/src/utils/useClipboard.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { AppState, Clipboard } from 'react-native'
+
+const CLIPBOARD_CHECK_INTERVAL = 1000 // 1sec
+
+export function useClipboard(): string {
+  const [clipboardContent, setClipboardContent] = useState('')
+
+  useEffect(() => {
+    let isMounted = true
+
+    async function checkClipboardContent() {
+      try {
+        const newClipboardContent = await Clipboard.getString()
+        if (!isMounted) {
+          return
+        }
+
+        setClipboardContent(newClipboardContent)
+      } catch (error) {
+        console.error('Error checking clipboard contents', error)
```

Comment:
Should be Logger right? Why didn't lint catch this?

Reply:
Good catch! Looks like we're allowing `console.error`